docs/tutorial.md

```diff
@@ -91,8 +100,21 @@ password: <password> # should look something like: XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 ```
 
-You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with
-username admin and password the action above.
+> NOTE: If you are using Multipass VM for this tutorial, you will need to route the IP from Multipass. To do this first get the IP of the Multipass VM.
+> Outside the Multipass VM run:
+> ```
+> multipass info my-juju-vm
+> ```
+> The IP you see here will be called <VM_IP> in this example.
+>
+> Then route:
+> ```
+> sudo ip route add <UNIT_IP> via <VM_IP>
+> ```
+
+
+You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with the admin username and password from the previous action.
```

Comment:
login --> log in
(login is a noun or adjective, while "log in" is a verb. I have to look this up constantly!)